src/main/java/data/Database.java

```diff
@@ -55,25 +60,25 @@ public OutputStream writeData(String runId, String type) {
     return null;
   }
 
+  /*Returns appropiate file name for data with this runId and type. */
   private String makeFileName(String runId, String type) {
-    return getDatabase() + "/" + runId + "_" + type;
+    return getDirectoryName() + "/" + runId + "_" + type;
   }
 
   /* Adds file to database given its runid and type. */
   public String newDatabaseEntry(String runId, String type) {
     String fileName = makeFileName(runId, type);
-    files.add(fileName);
     return fileName;
   }
 
   /* Return name of file with runid and type in database. */
   public String findName(String runId, String type) {
-    String fileName = getDatabase() + "/" + runId + "_" + type;
-    return files.get(0);
+    return makeFileName(runId, type);
```

Comment:
What's the point of findName isn't it redundant with makeFileName? Why not just make makeFileName public and call that directly?

Reply:
that makes sense, ill consolidate to file name method